tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
+			url:               "https://google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.google.co.in/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_uk.google.com_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.google.co.uk/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_www.bing.org_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.uk/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_uk.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.uk/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_in.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.in/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
 			shouldBeReachable: true,
 		},
 		{
-			name:              "REACHABLE_de.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://de.wikipedia.org/wiki/Wikipedia:Hauptseite",
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
```

Comment:
I didn't realize that we had wikipedia here!! So it has been working reliably so far? we haven't seen any flakes on this piece. In that case, please put this back as this tests that we can use virtual services to override service entries with TLS ports

Reply:
I restored all wikipedia tests :). they seem to work fine